pycqed/measurement/measurement_control.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, name: str,
     # Functions used to control the measurements #
     ##############################################
 
-    def run(self, name=None, mode='1D', **kw):
+    def run(self, name=None, exp_metadata=None, mode='1D', **kw):
         '''
```

Comment:
Could you add a docstring and type spec for the exp_metadata argument (and maybe also for the other input arguments while you are at it?)